src/recon_buildblock/ML_estimate_component_based_normalisation.cxx

```diff
@@ -3,6 +3,7 @@
     Copyright (C) 2019-2020, 2022, University College London
     Copyright (C) 2016-2017, PETsys Electronics
     Copyright (C) 2021, Gefei Chen
+    Copyright (C) 2024
```

Comment:
who owns the copyright if your current work?

Reply:
I had to check, its actually me. Do you want my name here or blank?

Reply:
yours! And we'll need another letter then :-)